doc/apiary/iotagent.apib

```diff
@@ -71,7 +71,7 @@ For example, `http://127.0.0.1:8080/iot/`.
 # Group Configuration API
 
 ## Services [/services{?limit,offset,resource,apikey,device}]
-Services are the higher level in IoT Platform. When you manipulate a service, you use a Fiware-Service header with its name. Parameters apply to different operations.
+Group configuration for iotagents.
```

Comment:
group configuration -> configuration group

Reply:
fixed in 8ba00f67db1897dddfea5c1bc64bdd0d9ac18aeb